inc/class-cachify.php

```diff
@@ -863,70 +854,65 @@ public static function touch_comment( $new_status, $old_status, $comment ) {
 	}
 
 	/**
-	 * Generate publish hook for custom post types
+	 * Removes the post type cache if saved or updated
 	 *
-	 * @since   2.1.7  Make the function public
-	 * @since   2.0.3
+	 * @since
+	 * @change
```

Comment:
Was the change history removed by intention?

Despite `@change` not being a valid PHPdoc tag (even though this project makes excessive use of it), empty `@since` does not make any sense either.

```php
@since 2.0.3 [implicit: created]
@since 2.1.7 Make the function public
```

perfectly points out relevant changes to this method...